dask/array/tests/test_reductions.py

```diff
@@ -198,10 +204,13 @@ def test_reduction_errors():
 
 
 @pytest.mark.slow
-@pytest.mark.parametrize("dtype", ["f4", "i4"])
+@pytest.mark.parametrize("dtype", ["f4", "i4", "c8"])
 def test_reductions_2D(dtype):
-    x = np.arange(1, 122).reshape((11, 11)).astype(dtype)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", np.ComplexWarning)
+        x = (np.arange(1, 122) + 1j * np.arange(1, 122)).reshape((11, 11)).astype(dtype)
     a = da.from_array(x, chunks=(4, 4))
+    warnings.simplefilter("default", np.ComplexWarning)
```

Comment:
Don't think we should need this unless I'm missing something?

```suggestion
```

Reply:
You're right, I misunderstood how these filters work